### ISSUE TITLE
Effective result of presence call diverge from typing annotation.

### ISSUE THREAD

**Issue by vindemasi** [centrifugal/centrifuge-js#220]
Hello all,

**Describe the bug**
It seems type annotation for `subscription.presence()` diverge from the effective result.

**To Reproduce**
1. Call `subscription.presence()`
2. See a responce payload like:
 ```json 
  {
    "f7cfb659-8cda-4561-a1b2-bb8ecb173713": {
        "user": "1",
        "client": "f7cfb659-8cda-4561-a1b2-bb8ecb173713",
        "conn_info": {
            "id": "1",
            "name": "Someone"
        }
    }
}
```

**Expected behavior**
It should return (as per type annotation) a property `connInfo` instead of `conn_info`.

**Versions**
 - OS: macOS
 - Browser
 - centrifuge-js v3.1.0
 - Server version 4.1.2

**Comment by FZambia** [comment]
Hello, thanks for opening!

Unfortunate thing, will fix keeping current field too - to not break someone's code.

**Comment by FZambia** [comment]
Fixed in 3.1.1
